testing/kfctl/scripts/create_existing_cluster.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Can the functionality of this script be achieved by 

https://github.com/kubeflow/testing/blob/a841aa0b2b1f6b64e06f5ab2be9f6df2ba3b214e/py/kubeflow/testing/util.py#L166-L170

Reply:
It can, but it basically requires me to translate the gcloud command to a JSON request body, which is not straightforward.
We can leave this to a followup PR is it's more desirable to do it that way.
I also didn't find any other code using that function.